Content.Server/Dragon/DragonSystem.cs

```diff
@@ -98,7 +100,9 @@ public override void Update(float frameTime)
             if (comp.RiftAccumulator >= comp.RiftMaxAccumulator)
             {
                 Roar(uid, comp);
-                QueueDel(uid);
+                if(TryComp<DevourerComponent>(uid, out var devourComponent))
+                    _devour.EmptyStomach(uid, devourComponent);
+                _body.GibBody(uid);
```

Comment:
wouldnt gibbing already empty the stomach since it would raise the event that gets handled by the devour system?